service/src/main/java/com/commercetools/pspadapter/payone/mapping/MappingUtil.java

```diff
@@ -12,16 +12,23 @@
 import io.sphere.sdk.models.Reference;
 import io.sphere.sdk.payments.Payment;
 import io.sphere.sdk.types.CustomFields;
+import org.apache.commons.lang3.StringUtils;
+import org.javamoney.moneta.function.MonetaryUtil;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
+import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
-import java.util.Locale;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
```

Comment:
To prevent class collisions, improve reading and compile time i would try to avoid usage of wildcards.